test/In.ProjectEKA.HipServiceTest/DataFlow/DataFlowClientTest.cs

```diff
@@ -6,21 +6,31 @@ namespace In.ProjectEKA.HipServiceTest.DataFlow
     using System.Net.Http;
     using System.Threading;
     using System.Threading.Tasks;
+    using Bogus;
+    using HipService.DataFlow.CryptoHelper;
+    using In.ProjectEKA.DefaultHip.DataFlow;
     using In.ProjectEKA.HipService.DataFlow;
     using Builder;
+    using Moq;
+    using Moq.Protected;
+    using Org.BouncyCastle.Crypto;
+    using Org.BouncyCastle.Crypto.EC;
+    using Org.BouncyCastle.Crypto.Generators;
+    using Org.BouncyCastle.Crypto.Parameters;
+    using Org.BouncyCastle.Security;
     using HipLibrary.Patient;
     using HipLibrary.Patient.Model;
     using Hl7.Fhir.Model;
-    using Microsoft.Extensions.DependencyInjection;
-    using Moq;
-    using Moq.Protected;
     using Optional;
     using Xunit;
     using Task = System.Threading.Tasks.Task;
 
     [Collection("Queue Listener Tests")]
     public class DataFlowClientTest
     {
+        private readonly Collect collect = new Collect("observation.json");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field 'collect'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5073772)